src/main/java/ch/uzh/ifi/hase/soprafs21/entities/SchedulingSession.java

```diff
@@ -22,30 +22,40 @@
     private Activity chosenActivity;
 
     @ElementCollection
-    @CollectionTable(name="SCHEDULING_SESSION_LOCATION_LIST", joinColumns = @JoinColumn(name = "locationId"))
+    @CollectionTable(name = "SCHEDULING_SESSION_LOCATION_LIST", joinColumns = @JoinColumn(name = "locationId"))
     @Column
     private List<String> locationList;
 
     @Column
     private String chosenLocation;
 
     @ElementCollection
-    @CollectionTable(name="SCHEDULING_SESSION_DATE_LIST", joinColumns = @JoinColumn(name = "dateId"))
+    @CollectionTable(name = "SCHEDULING_SESSION_DATE_LIST", joinColumns = @JoinColumn(name = "dateId"))
     @Column
     private List<LocalDateTime> dateList;
 
     @Column
     private LocalDateTime chosenDate;
 
-    public Long getId() {return id;}
+    public Long getId() {
```

Comment:
Not a big issue, but since we said we want to have the setters and getters in one line, maybe you could make  these ones also one line? Also those on the other files?

Reply:
strange, because it is on 1 line on my local branch.

Reply:
okay, maybe we can then ignore this